{"timestamp": "2017-09-21T08:15:02.527091Z", "level": "CRITICAL", "message": "Unhandled exception: %s", "system": "Windows 10", "gh_handle": "Kamokamocsai", "command": "C:\\Users\\Asus\\AppData\\Local\\Programs\\Python\\Python36-32\\Scripts\\gf start https://github.com/greenfox-academy/teaching-materials/blob/master/workshop/recursion/python.md", "feature": "START", "workshop_url": "https://github.com/greenfox-academy/teaching-materials/blob/master/workshop/recursion/python.md", "language": null, "type": "<class 'KeyError'>", "value": "Exception: 'repository'", "traceback": "File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\runpy.py\", line 193, in _run_module_as_main\n    \"__main__\", mod_spec)\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"C:\\Users\\Asus\\AppData\\Local\\Programs\\Python\\Python36-32\\Scripts\\gf.exe\\__main__.py\", line 9, in <module>\n    sys.exit(main())\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\gf\\app_starter.py\", line 15, in main\n    app_controller.run()\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\gf\\app_controller.py\", line 11, in run\n    task_check(get_command_argument())()\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\gf\\app_controller.py\", line 43, in start\n    start_service.main(get_filename())\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\gf\\start\\start_service.py\", line 13, in main\n    finish(workshop_data)\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\gf\\start\\start_service.py\", line 51, in finish\n    github.generate_issues(get_ex_names(workshop_data))\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\external\\web\\github.py\", line 98, in generate_issues\n    repo = get_repo(json_handler.get_working_repo())\n  File \"c:\\users\\asus\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\external\\local\\json_handler.py\", line 24, in get_working_repo\n    return get_config_json()[\"credentials\"][\"github\"][\"repository\"]"}
